src/Editor/components/Connections/components/Connection.tsx

```diff
@@ -18,35 +18,19 @@ export const ConnectionTrack: React.FC<{ output: Output; node: Node }> = ({ outp
   const svgOffset = useRecoilValue(svgOffsetState)
   const nextNode = nodes.find((node) => node.id === output.nextNodeId)
 
-  const outputPosition = useMemo(
-    () =>
-      node.rectPosition
-        ? {
-            x:
-              -svgOffset.x +
-              node.position.x +
-              output.position.x +
-              (styleConfig?.point?.width || DEFAULT_POINT_SIZE) / 2,
-            y:
-              -svgOffset.y +
-              node.position.y +
-              output.position.y +
-              (styleConfig?.point?.height || DEFAULT_POINT_SIZE) / 2
-          }
-        : node.position,
-    [node, svgOffset, styleConfig, output]
-  )
+  if (!nextNode) return null
 
-  const inputPosition = useMemo(
-    () =>
-      nextNode && {
-        x: -svgOffset.x + nextNode.position.x + (nextNode?.inputPosition?.x || 0),
-        y: -svgOffset.y + nextNode.position.y + (nextNode?.inputPosition?.y || 0)
-      },
-    [svgOffset, nextNode]
-  )
+  const outputPosition = node.rectPosition
+    ? {
+        x: -svgOffset.x + node.position.x + output.position.x + (styleConfig?.point?.width || DEFAULT_POINT_SIZE) / 2,
```

Comment:
if point exist in `styleConfig`, it must have width, that's why `?.width` and `?.height` not needed here

Reply:
It's not my task